src/Helpers/Translation/TranslationServiceProvider.php

```diff
@@ -93,11 +99,12 @@ public function getTranslator(string $locale): GettextTranslator
 
     /**
      * @param string $locale
+     * @param string $name
      * @return string
      */
-    protected function getFile(string $locale): string
+    protected function getFile(string $locale, string $name = 'default'): string
```

Comment:
It would be nice if you could also extend the unit tests to load multiple files

Reply:
Done